traits/trait_types.py

```diff
@@ -3336,6 +3338,144 @@ def as_ctrait(self):
         """
         return self.trait_maker.as_ctrait()
 
+# -------------------------------------------------------------------------------
+#  'None' trait:
+# -------------------------------------------------------------------------------
+
+
+class NoneTrait(TraitType):
```

Comment:
One more thing here - should probably have tests, even though it is currently not exposed outside of this module.

Reply:
Removed